packages/react-storage/package.json

```diff
@@ -58,6 +58,11 @@
     "react": "^16.14.0 || ^17.0 || ^18.0",
     "react-dom": "^16.14.0 || ^17.0 || ^18.0"
   },
+  "devDependencies": {
+    "jest-tsd": "^0.2.2",
+    "@tsd/typescript": "^5.1.6",
+    "@types/node": "^18.19.50"
```

Comment:
`jest-tsd` relying on node types, the workspace has had lower version of `@types/node` transitively, but versions lower. than `18.19.5` causes a typescript error while running the test. So using `"@types/node": "^18.19.50"` here.